deployment-aws/puppet/mevapi/manifests/cloudwatch_agent.pp

```diff
@@ -0,0 +1,48 @@
+class mevapi::cloudwatch_agent () {
+
+  if $mevapi::platform == 'aws' {
+
+    $cloudwatch_agent_dir = '/opt/aws/amazon-cloudwatch-agent'
+
+    file { '/tmp/amazon-cloudwatch-agent.deb':
+        ensure => present,
+        source => 'https://s3.amazonaws.com/amazoncloudwatch-agent/ubuntu/amd64/latest/amazon-cloudwatch-agent.deb',
+        mode   => '0550',
+        owner  => $mevapi::app_user
+    }
+
+    file { '/tmp/amazon-cloudwatch-agent.deb.sig':
```

Comment:
Is this actually used? I don't see it referenced anywhere in the code.

Reply:
i left it there in case we implemented some signature checking. is it worth it?

Reply:
YAGNI :) if you're not doing it now then it's not worth adding this just in case